draft-ietf-taps-interface.md

```diff
@@ -1763,6 +1766,20 @@ lower-bound rate below which the application does not deem
 a data transfer useful. It is given in bits per second.
 
 
+### Parallel Use of Multiple Paths
+
+Type:
+: Boolean
```

Comment:
Why Boolean? This is a selection property and should be a Preference as most others

Reply:
Why limit this to a selection property? FWIW, an application could turn this on or off later. Not in minset, but in the real MPTCP API:  https://datatracker.ietf.org/meeting/96/materials/slides-96-mptcp-4

Reply:
There can be different levels of "aggressiveness" in using multiple paths; anything actually using MPTCP or QUIC (migration or multipath) will likely need more than just a bool to set this meaningfully.

Reply:
TL;DR: can't we just keep the boolean for now, to have minset addressed, and discuss the rest later?  Because that may not be an easy one.

The longer version:
Ohhhhh.... I feel like I opened a gate that I shouldn't have opened, by pointing at the beyond-the-standards API slides. I mean, the thing that I'm guessing you have in mind (duplicating data across multiple paths to minimize latency vs. not doing that) really needs full control of subflows, which this API offers, but the (proposed) standard doesn't. Hence, we ended up with only the "disable" boolean in RFC 8303, and then minset - the absolute minimum that an application really needs, I guess.

As a way to keep the API simple, rather than adding more "aggression levels" or something like that, wouldn't it be better to let the application be in control, by using multiple TAPS connections as these subflows? We could maybe let it express a wish to couple the congestion control of connections if possible (because I'm guessing that, for the data duplication use case, that's the only thing that MPTCP buys you here? And... is that even a win?).

Reply:
@tfpauly I tend to rather have a "Do I want Multipath" preference in the first place (like the one @mwelzl proposed) and have the aggressiveness either controlled by an additional enum or by an Intents layer on top

Reply:
FWIW I agree that there should be different levels, like "no multipath", "only as backup path", and "balance load" or something. But I'm fine with keeping this a Boolean to start with and discuss later.

Reply:
@theri really boolean (as in the original version) or preference (which would match the other selection properties)?